diary/logdb.py

```diff
@@ -69,6 +69,10 @@ def close(self):
         except sqlite3.ProgrammingError:
             pass
 
+    def destroy(self):
```

Comment:
The build works on my computer, I'll look into the issue Travis CI is having.

I'm getting the feeling that you don't really want me contributing to the project. You seemed excited about open source, as am I, but you've been very hostile to my changes and haven't been responsive to my requests for comments / concerns. Please tell me if I am misguided in this assessment.